lib/primitives/base-element.js

```diff
@@ -114,13 +111,13 @@ module.exports = function(registry) {
       var attributes = {};
 
       this.attributes.keys().forEach(function (key) {
-        this.convertKeyToRefract(attributes, key, this.attributes.get(key));
+        attributes[key] = this.convertKeyToRefract(key, this.attributes.get(key));
       }, this);
 
       return attributes;
     },
 
-    fromRefract: function(doc) {
+    fromRefract: function(doc, options) {
```

Comment:
Naming question: is there a better name for this? This is not really for options for the functionality, but rather an object to merge into the newly-created element.